test/random_aplha_numeric_test.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:matomo/random_alpha_numeric.dart';
+
+void main() {
+  test('test randomAlphaNumeric returns string with correct length', () {
+    final randomString = randomAlphaNumeric(10);
+    expect(randomString.length, 10);
+  });
+
+  test('test randomAlphaNumeric return string with allowed characters', () {
+    final randomString = randomAlphaNumeric(10);
```

Comment:
Could we change the length here to be something else than 10, to make sure we're not just lucky with 10 ;)

Reply:
Changed the length to 42 ad 64. Hope that's what you mean.